docs/orchestrating-elastic-stack-applications/elasticsearch/advanced-node-scheduling.asciidoc

```diff
@@ -245,6 +245,14 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.annotations['topology.kubernetes.io/zone']
+        // The initContainers section is only required if using secureSettings in conjunction with zone awareness.
+        // initContainers:
+        // - name: elastic-internal-init-keystore
+        //   env:
+        //   - name: ZONE
+        //     valueFrom:
+        //       fieldRef:
+        //         fieldPath: metadata.annotations['topology.kubernetes.io/zone']
```

Comment:
Comments should begin with `#` 
```suggestion
        # The initContainers section is only required if using secureSettings in conjunction with zone awareness.
        # initContainers:
        # - name: elastic-internal-init-keystore
        #   env:
        #   - name: ZONE
        #     valueFrom:
        #       fieldRef:
        #         fieldPath: metadata.annotations['topology.kubernetes.io/zone']
```

nit: I would also move the `initContainers` section before the `containers` (right after `specs`)

Reply:
Thanks, I just accepted what my ide gave me here.   I also moved the section.